src/app/Models/Post.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Post extends Model
+{
+    use HasFactory;
+
+    protected $fillable = ['content', 'user_id' , 'image'];
```

Comment:
hmm can you add a relationship to the user table

Reply:
I have resolved this comment, sir. Thank you! 🙇